### PR TITLE
KHADRI-1 enum,optional,converter used

### PR DESCRIPTION
**Pull request by vikram-8790**
please verify and approve sir

### CODE REVIEW

File: resources/hibernate.cfg.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (khadri2021) [inline resources/hibernate.cfg.xml:1]: dont use xml configurations use proprties file and construct Factory object with Bootstrap API.

File: src/main/java/com/infotech/client/SaveDataClientTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import org.hibernate.Session;
+
+import com.infotech.entities.Address;
+import com.infotech.entities.AddressType;
+import com.infotech.entities.Employee;
+import com.infotech.entities.PhoneType;
+import com.infotech.util.HibernateUtil;
+
+public class SaveDataClientTest {
+
```
Comment (khadri2021) [inline src/main/java/com/infotech/client/SaveDataClientTest.java:15]: dont use Test names

File: src/main/java/com/infotech/entities/Address.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+@Entity
+@Table(name="address_table")
+public class Address {
+
+	@Id
+	@Column(name="adress_id")
+	@GeneratedValue(strategy=GenerationType.IDENTITY)
+	private Integer addressId;
+	
+	@Column(name = "street_name",length=50)
```
Comment (khadri2021) [inline src/main/java/com/infotech/entities/Address.java:23]: format it properly

Comment (khadri2021) [reply src/main/java/com/infotech/entities/Address.java:23]: use ctrl+shift+F

File: src/main/java/hibernate.cfg.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (khadri2021) [inline src/main/java/hibernate.cfg.xml:1]: duplicated , try to avoid

File: resources/hibernate.cfg.xml (hunk 1 above)
Comment (vikram-8790) [reply resources/hibernate.cfg.xml:1]: i have changed successfully

File: src/main/java/hibernate.cfg.xml (hunk 4 above)
Comment (vikram-8790) [reply src/main/java/hibernate.cfg.xml:1]: i removed

File: src/main/java/com/infotech/client/SaveDataClientTest.java (hunk 2 above)
Comment (vikram-8790) [reply src/main/java/com/infotech/client/SaveDataClientTest.java:15]: ok i changed

File: src/main/java/com/infotech/entities/Address.java (hunk 3 above)
Comment (vikram-8790) [reply src/main/java/com/infotech/entities/Address.java:23]: ok done

**Pull request closed without merging**